examples/live/generate_injections.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import os, sys
+from pycbc.io import record
+from pycbc.inject import InjectionSet
+
+if os.path.exists('./test_inj1.hdf'):
+    raise OSError("output-file 1 already exists")
+    
+if os.path.exists('./test_inj2.hdf'):
+    raise OSError("output-file 2 already exists")
+
+    
+# injection 1    
+static_params = { 'f_lower': 18.0, 'f_ref': 18.0, 'approximant': 'SEOBNRv4', \
```

Comment:
Please remove the backslashes, as they are not required when you have a list or dict.